python/cucim/src/cucim/core/operations/spatial/rotate_and_flip.py

```diff
@@ -130,7 +131,8 @@ def image_rotate_90(
 def rand_image_flip(
     img: Any,
     spatial_axis: tuple(),
-    prob: float = 0.1
+    prob: float = 0.1,
```

Comment:
What would be the good default value for this?
Having `0.5` for the default value looks more useful here?

Reply:
Will update it.